app/src/substrate_dispatch_V8.c

```diff
@@ -6315,6 +6315,7 @@ bool _getMethod_IsNestingSupported_V8(uint8_t moduleIdx, uint8_t callIdx)
     uint16_t callPrivIdx = ((uint16_t)moduleIdx << 8u) + callIdx;
 
     switch (callPrivIdx) {
+    case 1: // System:Remark
```

Comment:
If you verify the very end of the switch statement, including this line will remove system:remark from nesting supported list. Every method listed here will return false. I think that is the opposite that you want.